app/js/dialog.js

```diff
@@ -0,0 +1,8 @@
+jQuery(function($) {
+  $('#privacyModal')
+    .modal({backdrop: 'static', keyboard: false})
+    .modal('show')
+    .on('hide.bs.modal', function () {
+      document.cookie = "privacy_modal_showed=1; path=/; domain=" + privacy_modal_hostname;
```

Comment:
Why the explicit domain setting? This makes it harder to achieve https://github.com/silverstripe/bambusa-installer/pull/15, because I need to set the global via a PHP controller there as well. https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#Write_a_new_cookie says that you don't need the domain - I'm assuming the domain from `document.location.host` would match `Director::host()`?

Reply:
I've removed that bit.